lib/elixir/lib/exception.ex

```diff
@@ -956,14 +959,17 @@ defmodule MismatchedDelimiterError do
         end_line: end_line,
         end_column: end_column,
         description: description,
+        opening_delimiter: _open_delimiter,
+        closing_delimiter: _close_delimiter,
+        expected_closing_delimiter: expected_closing_delimiter,
```

Comment:
Nitpick: instead of storing it here, expose `:elixir_tokenizer.terminator` and call it from here. :)